backend/src/models/Post.js

```diff
@@ -39,5 +39,6 @@ module.exports = {
     default: () => new Date().toISOString(),
   },
   language: { type: 'string', allow: [null] },
+  blurImage: { type: 'boolean', default: false },
```

Comment:
```suggestion
  imageBlurred: { type: 'boolean', default: false },
  imageBlurredByModerator: { type: 'boolean', default: false },
```

Reply:
talking about it, it would be nice to keep the image properties in the database start with `image`

Reply:
ok

Reply:
I changed the database name from "blurImage" to "imageBlurred".